src/Components/ArtistFilter/Artworks.tsx

```diff
@@ -73,22 +98,38 @@ class Artworks extends React.Component<Props> {
   }
 }
 
-export default createPaginationContainer(
+export default createRefetchContainer(
   Artworks,
   {
     filtered_artworks: graphql`
       fragment Artworks_filtered_artworks on FilterArtworks
         @argumentDefinitions(
-          count: { type: "Int", defaultValue: 10 }
-          cursor: { type: "String", defaultValue: "" }
+          first: { type: "Int", defaultValue: 10 }
+          after: { type: "String", defaultValue: "" }
         ) {
         __id
-        artworks: artworks_connection(first: $count, after: $cursor)
-          @connection(key: "Artworks_filtered_artworks") {
+        artworks: artworks_connection(first: $first, after: $after) {
           pageInfo {
             hasNextPage
             endCursor
           }
+          pageCursors {
```

Comment:
I didn't do it in this PR, but the pagination stuff can neatly go into its own component/fragment container.